content/rs/getting-started/getting-started-active-active.md

```diff
@@ -109,13 +109,12 @@ Now we have two Redis Enterprise Software clusters with FQDNs
 
     ![crdb-activate](/images/rs/crdb-activate.png)
 
-    <!-- Also in getting-started-docker.md -->
-    Note: If you cannot activate the database because of a memory limitation,
-    make sure that Docker has enough memory allocated in the Advanced section
-    of Docker Settings.
+    {{< note >}}
+    {{< embed-md "docker-memory-limitation.md" >}}
+    {{< /note >}}
 
-1. After the Active-Active database is created, you can now visit each cluster 1 at
-https://localhost:8443 and cluster 2 at https://localhost:8445.
+1. After the Active-Active database is created, access the RS admin console
```

Comment:
If there's no other admin console, why do we need to quality this with "RS"?

Reply:
I would treat it as a product name, but I can hear the argument against also.
Should it be Admin Console or admin console?